src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -146,148 +136,6 @@ const ShareTrajectoryModal = ({
         </>
     );
 
-    /** Embedded Snippet */
-    interface EmbedSettings {
-        width: number;
-        height: number;
-        userProvidedProportions: boolean;
-    }
-
-    const defaultEmbedSettings: EmbedSettings = {
-        width: 834,
-        height: 480,
-        userProvidedProportions: false,
-    };
-
-    const trajectory = getUrlParamValue(
-        window.location.href,
-        URL_PARAM_KEY_FILE_NAME
-    );
-    const [embedSettings, setEmbedSettings] =
-        React.useState(defaultEmbedSettings);
-    const [showEmbedSettingsPanel, setShowEmbedSettingsPanel] =
-        React.useState(false);
-
-    const generateEmbedSnippet = (): string => {
-        const { width, height } = embedSettings;
-
-        const snippetHeight =
-            embedSettings.userProvidedProportions && embedSettings.height
-                ? height
-                : defaultEmbedSettings.height;
-        const snippetWidth =
-            embedSettings.userProvidedProportions && embedSettings.width
-                ? width
-                : defaultEmbedSettings.width;
-        const url = `https://simularium.allencell.org/embed?trajFileName=${trajectory}&t=0`;
-        return `<iframe height="${snippetHeight}" width="${snippetWidth} "src="${url}" title="Simularium" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>`;
-    };
-
-    const [embedSnippet, setEmbedSnippet] = React.useState(
-        generateEmbedSnippet()
-    );
-
-    useEffect(() => {
-        setEmbedSnippet(generateEmbedSnippet());
-    }, [embedSettings]);
-
-    const handleEmbeddedSizeInputs = (
-        setting: "height" | "width",
-        input: string
-    ) => {
-        const numericInput = input ? parseInt(input) : 0;
-        let roundedLimitedInput;
-
-        if (setting === "width") {
-            roundedLimitedInput = numericInput >= 3840 ? 3840 : numericInput;
-        } else if (setting === "height") {
-            roundedLimitedInput = numericInput >= 2160 ? 2160 : numericInput;
-        }
-
-        setEmbedSettings({ ...embedSettings, [setting]: roundedLimitedInput });
-    };
```

Comment:
I said it above, but this spread, and changing the value seems to take away the simplicity of just `setWidth` and `setHeight`